arch/Kconfig

```diff
@@ -351,42 +351,42 @@ config ARCH_HAS_THREAD_ABORT
 # Hidden PM feature configs which are to be selected by
 # individual SoC.
 #
-config HAS_STATE_LOW_POWER_1
+config HAS_SLEEP_STATE_1
```

Comment:
HAS_SLEEP_STATE_X matches with SYS_POWER_STATE_SLEEP_X.
Could we have the same order of "STATE" and "SLEEP" in both?

Reply:
This looks good. Then I would suggest HAS_SYS_POWER_{DEEP_}SLEEP_STATE_X as a matching option.